gradle-sls-packaging/src/main/groovy/com/palantir/gradle/dist/service/tasks/LaunchConfigTask.groovy

```diff
@@ -39,6 +39,7 @@ class LaunchConfigTask extends DefaultTask {
             '-XX:+UseGCLogFileRotation',
             '-XX:GCLogFileSize=10M',
             '-XX:NumberOfGCLogFiles=10',
+            '-XX:ErrorFile=var/log/hs_err_pid%p.log',
```

Comment:
Actually not sure this really belongs in the `gcJvmOpts` list (we concatenate tmpdirJvmOpts + gcJvmOpts + dnsJvmOpts + defaultJvmOpts) below.  Maybe worth a new list like `crashJvmOpts`?

Reply:
Also looking at the master list of options: https://www.oracle.com/technetwork/articles/java/vmoptions-jsp-140102.html, perhaps we should make the heap dumps not land in the SLS root either (`-XX:HeapDumpPath=./java_pid<pid>.hprof`)?

Reply:
Yeah was wondering that myself and wasn't sure if I should put it in a dedicated variable. I moved it to loggingJvmOpts and amended the commit. Will make a separate commit for the heap dumps.

Reply:
Actually, did you mean `-XX:HeapDumpPath=var/log/java_pid%p.hprof`?